pkg/utils/predicate.go

```diff
@@ -159,6 +159,7 @@ func (p OwnersOwnerInNamespacePredicate) ownersOwnerInNamespace(ownerRefs []meta
 func NewOwnersOwnerInNamespacePredicate(client client.Client) OwnersOwnerInNamespacePredicate {
 	return OwnersOwnerInNamespacePredicate{
 		client: client,
+
```

Comment:
Why though?

Reply:
It fixes the linters 🤷‍♂️

Reply:
Is that instead of aligning the arguments?

Reply:
We use two linters that have a different opinion on aligning of the arguments... I'm surprised we haven't seen this before

Reply:
![this is fine](https://media.giphy.com/media/NTur7XlVDUdqM/giphy.gif)

alrighty then, carry on.